core/src/main/scala/com/salesforce/op/dsl/RichMapFeature.scala

```diff
@@ -1026,15 +1028,16 @@ trait RichMapFeature {
       blackListKeys: Array[String] = Array.empty,
       trackNulls: Boolean = TransmogrifierDefaults.TrackNulls,
       others: Array[FeatureLike[EmailMap]] = Array.empty,
-      maxPctCardinality: Double = OpOneHotVectorizer.MaxPctCardinality
+      maxPctCardinality: Double = OpOneHotVectorizer.MaxPctCardinality,
+      cleanTextParams: CleanTextParams = CleanTextParams(true, false)
```

Comment:
How are we going to evolve these params, keep adding booleans? should this be a list of enum values with corresponding text transformers?